docs/ref/models/querysets.txt

```diff
@@ -1771,7 +1772,7 @@ whenever a request to a page has a side effect on your data. For more, see
 
 A convenience method for updating an object with the given ``kwargs``, creating
 a new one if necessary. The ``defaults`` is a dictionary of (field, value)
-pairs used to update the object.
+pairs used to update the object. The ``value``s in ``defaults`` can be callables.
```

Comment:
Please add a `.. versionchanged:: 1.11` annotation as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features, as well as a mention in the 1.11 release notes.